pyproject.toml

```diff
@@ -70,4 +70,5 @@ black = ""
 pytype = "^2022.1.13"
 pytest = ""
 pytest-parallel = ""
-pytest-check = ""
\ No newline at end of file
+pytest-check = ""
+pytest-xdist = "^2.5.0"
```

Comment:
Wait, why do we need that? We already have pytest-parallel

Reply:
pytest-parallel has a weird behaviour on the arm linux machines where we plan to run semantic tests, where it just randomly stops spawning workers once its done n-1 workers